### ISSUE TITLE
The context menu is not accessible to users navigating with a keyboard

### ISSUE THREAD

**Issue by dariiing** [cryptpad/cryptpad#1631]
Users are expected to right-click on documents and folders to access context-specific options. However, this functionality is not accessible via keyboard alone. This causes users who rely on keyboard navigation to be unable to access the functionality that is available through
right-clicking. To address this issue, an alternative keyboard-accessible method should be provided for accessing the context menu.
